earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/common/utils.py

```diff
@@ -328,9 +328,17 @@ def RunCmd(os_cmd):
   try:
     if isinstance(os_cmd, str):
       os_cmd = shlex.split(os_cmd)
-    results = subprocess.check_output(os_cmd)
+    p = subprocess.Popen(os_cmd, shell=False,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE)
+    results = p.communicate()[1]
+    return_code = p.returncode
+    if return_code != 0:
+      errorMessage = "%s (return code %d)" % (results, return_code)
```

Comment:
Style: Using string's `format` method is recommended over the `%` operator nowadays. I find it can be, also, more readable.

Reply:
done